components/script/dom/node.rs

```diff
@@ -349,7 +354,16 @@ impl Node {
             kid.teardown();
         }
     }
+    /// method on Node that returns Vec<Root<MutationObserver>>
+    pub fn registered_mutation_observers_for_type(&self) -> &DOMRefCell<Vec<MutationObserver>> {
```

Comment:
Given that we don't pass any kind of type argument here, let's rename this to `registered_mutation_observers`.